tardis/utilities/utils.py

```diff
@@ -98,7 +98,9 @@ def slurm_cmd_option_formatter(options: AttributeDict) -> str:
         except AttributeError:
             pass
         else:
-            if option_string:  # add additional space between short and long options
+            # add additional space between short and long options if there are long
+            # options
+            if option_string and "long" in options and options["long"] != {}:
```

Comment:
Could you revert this change, please? Instead you can return `option_string.strip()` in Line 107. Since I think this is much cleaner.